docs/reference/commandline/secret_inspect.md

```diff
@@ -60,24 +60,21 @@ $ docker secret inspect secret.json
         "CreatedAt": "2016-10-27T23:25:43.909181089Z",
         "UpdatedAt": "2016-10-27T23:25:43.909181089Z",
         "Spec": {
-            "Name": "secret.json",
-            "Data": null
```

Comment:
Note that when I ran the command, it didn't actually display that the data was null (it was just omitted)